packages/bpk-component-image/src/withLazyLoading.js

```diff
@@ -124,6 +160,7 @@ export default function withLazyLoading(Component, document) {
         <div
           id={this.placeholderReference}
           ref={element => {
+            // $FlowFixMe
```

Comment:
Flow error
```
[flow] null (This type is incompatible with HTMLElement)
```
flow is inferring the HTMLElement like this
```element: null | HTMLDivElement```
ignoring my type definition, any idea what's going on?

Reply:
Which type definition?

Reply:
https://github.com/Skyscanner/backpack/pull/431/files/6458d11cd9db3bc265558c454452a468f59a3a32#diff-be182dff2185fb989cfd339d396e50dfR42

Reply:
You should null check it before assigning then so Flow can refine the type. See the [docs](https://flow.org/en/docs/types/maybe/#toc-refining-maybe-types)

Reply:
Flow has it's own way to deal with refs https://flow.org/en/docs/react/refs/

Reply:
Actually yeah that makes more sense. It should be `?HTMLElement` in the class since it will initially be null